cmd/testworkflow-init/main.go

```diff
@@ -369,7 +376,18 @@ func main() {
 				stopTimeoutWatcher := orchestration.WatchTimeout(finalizeTimeout, leaf...)
 
 				// Run the command
-				commands.Run(*action.Execute, currentContainer)
+				d := data.GetState()
+				recorderConfig := utilization.Config{
+					Dir:       "./.tk/metrics",
+					Skip:      action.Execute.Toolkit,
+					Workflow:  d.InternalConfig.Workflow.Name,
+					Step:      step.Ref,
+					Execution: d.InternalConfig.Execution.Id,
```

Comment:
Maybe be worth to consider using `Resource.Id` too - it will differ on parallel steps and services